admin/app/models/user.js

```diff
@@ -51,4 +51,7 @@ export default class User extends Model {
   get participationsCount() {
     return this.participations.length;
   }
+  get connexionMethodsCount() {
+    return this.username && this.email ? this.authenticationMethods.length + 1 : this.authenticationMethods.length;
+  }
```

Comment:
```suggestion
  get authenticationMethodCount() {
    return this.username && this.email ? this.authenticationMethods.length + 1 : this.authenticationMethods.length;
  }
```

Justification : 

Je pense qu'il faut garder le mot `authenticationMethod` partout dans le code. Le mot « connexion » est la notion qui est présentée aux utilisateurs, même si au final ce `authenticationMethodCount` n'est pas toujours égale à `this.authenticationMethods.length`, ce qui est au premier abord est un peu perturbant et fait s’interroger si le modèle de données de la table `authenticationMethods` est vraiment bien adapté.

Ensuite en anglais on dit « authenticationMethod count » (sans « s ») et pas « authenticationMethod**s** count », cf. https://en.wiktionary.org/wiki/bit-count_integrity, https://en.wiktionary.org/wiki/body_count, 

Idéalement, pour éviter que cette erreur de nommage prolifère, il faudrait ajouter un commit de correction des noms de variables mal nommées : 

`organizationMembershipsCount`→`organizationMembershipCount`
`certificationCenterMembershipsCount`→`certificationCenterMembershipCount`
`participationsCount`→`participationCount`